src/components/Button/Button.scss

```diff
@@ -158,6 +158,7 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
 
 .primary {
   --p-button-color: var(--p-primary-action);
+  --p-button-text: var(--p-text-on-primary);
```

Comment:
It would be really useful if we could make `textOnPrimary` use the `dark` config when `primaryAction` is dark, and use the `light` config when `primaryAction` is light. That would open up `Primary` to be user configurable while still preserving accessibility.

I think that's a job for another PR though, as it will require some code changes to `utils.ts`

cc: @tmlayton